src/script/externalRoute.ts

```diff
@@ -20,95 +20,46 @@
   }
   return undefined;
 };
 
 export const getAccountPagesUrl = (path: string = ''): string | undefined => {
-  const accountPagesUrl = `${URL.ACCOUNT}${path}`;
-  return URL.ACCOUNT ? accountPagesUrl : undefined;
+  const accountPagesUrl = `${URL.ACCOUNT_BASE}${path}`;
+  return URL.ACCOUNT_BASE ? accountPagesUrl : undefined;
 };
 
-export const getPrivacyPolicyUrl = (): string => addLocaleToUrl(URL.PRIVACY_POLICY || undefined);
-export const getTermsOfUsePersonalUrl = (): string => addLocaleToUrl(URL.TERMS_OF_USE_PERSONAL || undefined);
-export const getTermsOfUseTeamUrl = (): string => addLocaleToUrl(URL.TERMS_OF_USE_TEAMS || undefined);
+export const getPrivacyPolicyUrl = (): string | undefined => addLocaleToUrl(URL.PRIVACY_POLICY || undefined);
+export const getTermsOfUsePersonalUrl = (): string | undefined =>
+  addLocaleToUrl(URL.TERMS_OF_USE_PERSONAL || undefined);
+export const getTermsOfUseTeamUrl = (): string | undefined => addLocaleToUrl(URL.TERMS_OF_USE_TEAMS || undefined);
 
-export const getManageServicesUrl = (utmSource?: string): string =>
-  getTeamSettingsUrl(URL_PATH.MANAGE_SERVICES, utmSource);
-export const getManageTeamUrl = (utmSource?: string): string => getTeamSettingsUrl(URL_PATH.MANAGE_TEAM, utmSource);
+export const getManageServicesUrl = (utmSource?: string): string | undefined =>
+  getTeamSettingsUrl(URL.SUBPATH.MANAGE_SERVICES, utmSource);
+export const getManageTeamUrl = (utmSource?: string): string | undefined =>
+  getTeamSettingsUrl(URL.SUBPATH.MANAGE_TEAM, utmSource);
 
-export const getCreateTeamUrl = (): string =>
-  Config.getConfig().FEATURE.ENABLE_ACCOUNT_REGISTRATION && `${Config.getConfig().URL.TEAMS_BASE}/register/email`;
-export const getDecryptErrorUrl = (): string => getHelpCenterUrl(URL_PATH.DECRYPT_ERROR_1);
-export const getPrivacyUnverifiedUsersUrl = (): string => getHelpCenterUrl(URL_PATH.PRIVACY_UNVERIFIED_USERS);
-export const getPrivacyWhyUrl = (): string => getHelpCenterUrl(URL_PATH.PRIVACY_WHY);
+export const getCreateTeamUrl = (): string | undefined =>
+  Config.getConfig().FEATURE.ENABLE_ACCOUNT_REGISTRATION ? `${URL.TEAMS_BASE}${URL.SUBPATH.CREATE_TEAM}` : undefined;
 
-export const addLocaleToUrl = (url?: string): string => {
+export const addLocaleToUrl = (url?: string | undefined): string | undefined => {
   if (!url) {
     return undefined;
   }
   const language = currentLanguage().slice(0, 2);
   const websiteLanguage = language == 'de' ? language : 'en';
   return url.replace(Config.getConfig().URL.WEBSITE_BASE, `${Config.getConfig().URL.WEBSITE_BASE}/${websiteLanguage}`);
 };
-
-const addLocaleToHelpCenterUrl = (url?: string): string => {
-  if (!url) {
-    return undefined;
-  }
-  const language = currentLanguage().slice(0, 2);
-  const websiteLanguage = language == 'de' ? language : 'en-us';
-  return url.replace(
-    `${Config.getConfig().URL.SUPPORT.INDEX}`,
-    `${Config.getConfig().URL.SUPPORT.INDEX}/hc/${websiteLanguage}`,
-  );
-};
```

Comment:
help center links do not need to be localized as they follow the browser locale setting